src/main/java/fr/alasdiablo/janoeo/config/OverworldConfig.java

```diff
@@ -20,50 +22,78 @@
 
     public static class Config {
 
+        public final HashMap<String, ForgeConfigSpec.BooleanValue> Ores = new HashMap<String, ForgeConfigSpec.BooleanValue>();
```

Comment:
+ Do not use HashMap as base variable type (use Map instead)
+ The key of this Map, make the mod less modular
+ `new HashMap<String, ForgeConfigSpec.BooleanValue>()` can be replaced by `new HashMap<>()` we already know the type
+ Variable name like a class ?